admin/controllers/school.js

```diff
@@ -19,9 +20,11 @@ controller.getSchoolLandingPage = async (req, res, next) => {
     const checkWindowData = await checkWindowV2Service.getActiveCheckWindow()
     const schoolName = await schoolService.findSchoolByDfeNumber(req.user.School)
     const featureEligibilityData = schoolHomeFeatureEligibilityPresenter.getPresentationData(checkWindowData, req.user.timezone)
+    const hasIncompleteChecks = await pupilRegisterService.hasIncompleteChecks(req.user.schoolId)
```

Comment:
This could have a performance impact on school home page. 
It will need to be measured with load testing.